styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+ color: blue; 
+}
```

Comment:
![image](https://user-images.githubusercontent.com/78251927/211651701-5dc44512-b71b-4cb0-8432-c5d584865fa8.png)

Please kindly fix the Linters / Stylelint error, all linter results should be green ✔️. AKA: Linter should report NO errors.

To view the report in your terminal, Run  `npx stylelint "**/*.{css,scss}"`.  then you can solve the error 

Or automatically fix by using `npx stylelint "**/*.{css,scss}"`